Swift/AEPSampleApp/AdIdUtils.swift

```diff
@@ -68,38 +60,32 @@ class AdIdUtils {
                 // Tracking authorization dialog was shown and authorization given
                 case .authorized:
                     // IDFA now accessible
-                    print("\(status)")
+                    print("Authorization status: \(status)")
                 // For all cases below (that is, not .authorized), IDFA is all-zeros
                 // Tracking authorization dialog was shown and permission is denied
                 case .denied:
-                    print("\(status)")
+                    print("Authorization status: \(status)")
                 // Tracking authorization dialog has not been shown
                 case .notDetermined:
-                    print("\(status)")
-                // Tracking authorization dialog is not allowed to be shown
+                    print("Authorization status: \(status)")
+                // Tracking authorization dialog is not allowed to be shown; not controlled by the user
                 case .restricted:
-                    print("\(status)")
+                    print("Authorization status: \(status)")
                 @unknown default:
-                    print("\(status)")
+                    print("Authorization status: \(status)")
                 }
+                callbackHandler()
```

Comment:
In the case of L77, because the `ATTrackingManager.requestTrackingAuthorization` is async, moving the callbackHandler call to the end of the function may cause the callback to be fired before the request flow completes. Should it be moved?

Reply:
No, makes sense, sorry I missed that was inside the ATTrackingManager callback.